mitosheet/src/components/taskpanes/ControlPanel/FilterTab/UniqueValuesCard.tsx

```diff
@@ -122,83 +127,35 @@ export function ValuesTab(
             setIsAllData(_uniqueValueObj.isAllData);
+            setToggledValues([]);
         } else {
             setUniqueValueCounts([])
         }
         setLoading(false);
     }
 
-
-    /* 
-        Helper function for getting the index of the UniqueValueCount in UniqueValueCounts from the index 
-        of the UniqueValueCount in the searchedUniqueValueCounts. It exploits the invariant that
-        these are _unique value_ counts. 
-    */
-    const getUniqueValueCountIndexFromSortedIndex = (index: number): number => {
-        const value = sortedUniqueValueCounts[index].value
-        return uniqueValueCounts.findIndex(uniqueValueCount => {
-            return uniqueValueCount.value === value
-        })
-    }
-
-    /*
-        Toggles the exclusive filter for a specific value. An exclusive filter is the NOT_EXACTLY filter condition
-        for strings, numbers, and dates. For booleans its either the IS_TRUE or IS_FALSE conditions. 
-
-        For a specific value in this column, the toggleExclusiveFilter determines if there is an exclusive filter that is excluding the value.
-        If there is, it removes it. If there's is not, it applies it. 
-    */
-    const toggleExclusiveFilters = (values: (string | number | boolean)[]): void => {
-        // Generate the filter
-        
-        props.setFilters((prevFilters) => {
-            let newFilters = [...prevFilters];
-
-            values.forEach(value => {
-                const exclusiveFilter = getExclusiveFilterData(props.columnDtype, value)
-                const originalFilterLength = newFilters.length
-
-                // Remove the filter if it exists
-                newFilters = newFilters.filter(filter => {
-                    return isFilterGroup(filter) || !areFiltersEqual(filter, exclusiveFilter)
-                })
-
-                // If the filter didn't exist, then add it. 
-                if (newFilters.length === originalFilterLength) {
-                    newFilters.push(exclusiveFilter)
-                }
-            })
-
-            return newFilters;
-        })
-    }
-
     const sortedUniqueValueCounts = sortUniqueValueCounts(uniqueValueCounts, sort);
     const disabledMessage = getFilterDisabledMessage(props.columnDtype);
 
     return (
         <Fragment>
             <Row justify='space-between'>
                 <Col flex='1' offsetRight={1}>
-                    <p className='text-header-2'> 
-                        Unique Values
-                    </p>
+                    <Row justify='start' align='center' suppressTopBottomMargin>
+                        <Col>
+                            <p className='text-header-2'> 
+                                Unique Values
+                            </p>
+                        </Col>
+                        <Col offset={1} title='Click to refresh unique values, in the case they are out of date.'>
+                            <div onClick={loadUniqueValueCounts}>
```

Comment:
Maybe we can make the onClick on the col, so its easier to click? I think this is why Jake struggled to get the ? icons to display the text. ... at least I was able to see the text on Brave.